sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -33,32 +49,65 @@ private void InitializeState()
         private void SetPingState()
         {
             _state.Update(LiveMetricsState.Ping);
+            _shouldCollect = false;
             _callbackAction = OnPing;
             _period = _pingPeriod;
+            _lastSuccessfulPing = DateTimeOffset.UtcNow;
+            _evaluateBackoff = () => DateTimeOffset.UtcNow - _lastSuccessfulPing > _maximumPingInterval;
         }
 
         private void SetPostState()
         {
             _state.Update(LiveMetricsState.Post);
+            _shouldCollect = true;
             _callbackAction = OnPost;
             _period = _postPeriod;
+            _lastSuccessfulPost = DateTimeOffset.UtcNow;
+            _evaluateBackoff = () => DateTimeOffset.UtcNow - _lastSuccessfulPost > _maximumPostInterval;
+        }
+
+        private void SetBackoffState()
+        {
+            _state.Update(LiveMetricsState.Backoff);
+            _shouldCollect = false;
+            _callbackAction = BackoffConcluded;
+            _period = _backoffPeriod;
+            _evaluateBackoff = () => false;
+        }
+
+        private void BackoffConcluded()
+        {
+            // when the backoff period is complete, we switch to Ping.
+            SetPingState();
```

Comment:
This would wait another `_pingPeriod` after the backoff period is complete, totally to 61 seconds. I'm not sure if the additional wait matters though.

Reply:
Good catch! I don't think it matters either, but it's something i'll make a note of.